### DISCUSSION TITLE
Images aren't showing in README of Dodge Charger Game

### DISCUSSION THREAD

**Discussion by muditchoudhary** [prathimacode-hub/Awesome_Python_Scripts#564] [Q&A]
# Problem
![Editing Awesome_Python_Scripts_README md at main · prathimacode-hub_Awesome_Python_Scripts - Google Chrome 29_06_2021 6_18_45 pm_LI](https://user-images.githubusercontent.com/74391865/123800495-fba72b00-d906-11eb-83f0-8c81d9eef1d2.jpg)
Images aren't showing because I placed wrong slash. Instead of forward-slash I put backward-slash and some emojis aren't also showing in readme file. 

# Question
I want to ask, do I have to open a new issue or I can update it directly?

**Comment by prathimacode-hub** [comment]
You can update it directly. @MuditCh57
